README.md

```diff
@@ -313,9 +341,16 @@ Cakan, C., Jajcay, N. & Obermayer, K. neurolib: A Simulation Framework for Whole
 
 Caglar Cakan (cakan@ni.tu-berlin.de)  
 Department of Software Engineering and Theoretical Computer Science, Technische Universität Berlin, Germany  
-Bernstein Center for Computational Neuroscience Berlin, Germany  
+Bernstein Center for Computational Neuroscience Berlin, Germany 
+
+For questions or comments on the optimal control module, please contact
+Lena Salfenmoser (lena.salfenmoser@tu-berlin.de)
+Department of Software Engineering and Theoretical Computer Science, Technische Universität Berlin, Germany  
+
```

Comment:
```suggestion
```

This should not be here. I suggest adding the citation to the OC paper below the OC example in the Readme.

Reply:
Removed. Not sure if citing the OC paper below the OC example makes sense, please have a look...